backend/src/main/java/ch/puzzle/okr/service/business/ActionBusinessService.java

```diff
@@ -74,4 +76,25 @@ public void deleteEntityById(Long id) {
     public void deleteEntitiesByKeyResultId(Long keyResultId) {
         getActionsByKeyResultId(keyResultId).forEach(action -> deleteEntityById(action.getId()));
     }
+
+    public List<Action> createDuplicateOfActions(KeyResult oldKeyResult, KeyResult newKeyResult) {
```

Comment:
```suggestion
    public List<Action> createDuplicates(KeyResult oldKeyResult, KeyResult newKeyResult) {
```
I would rename this, simply because the `Actions` part is already heavily implied by it being in the `ActionBusinessService` :smiley: